# Questão Proposta 

A Fig. 28-12 ilustra o princípio de funcionamento do espectrômetro de massa, um instrumento usado para medir a massa de íons. Um íon de massa m (a ser medida) e carga q é produzido na fonte S e acelerado pelo campo elétrico associado a uma diferença de potencial V. O íon entra em uma câmara de separação na qual existe um campo magnético uniforme B perpendicular à sua velocidade. O campo faz com que o íon descreva uma trajetória semicircular antes de atingir um detector situado na superfície inferior da câmara. Suponha que B = 80.000 mT, V = 1000,0 V e que íons de carga q = +1,6022 × 10⁻¹⁹ C atinjam o detector em um ponto situado a uma distância x = 1,6254 m do ponto de entrada na câmara. Qual é a massa m dos íons em unidades de massa atômica? (Eq. 1-7: 1 u = 1,6605 × 10⁻²⁷ kg).

In [2]:
# Cálculo da massa dos íons em um espectrômetro de massa
import math

# Dados fornecidos
B_mT = 80.000    # campo magnético em mT
B = B_mT * 1e-3  # converter para Tesla
V = 1000.0       # potencial em volts
q = 1.6022e-19   # carga do íon em C
x = 1.6254       # distância do detector a partir da entrada (m)

r = x / 2

# Cálculo da massa em kg: m = q * B^2 * r^2 / (2 * V)
m_kg = q * B**2 * r**2 / (2 * V)

# Converter para unidades de massa atômica (u)
u = 1.6605e-27  # kg
m_u = m_kg / u

print(f"Raio da trajetória (r): {r:.4f} m")
print(f"Massa do íon: {m_kg:.4e} kg")
print(f"Massa do íon: {m_u:.4f} u")


Raio da trajetória (r): 0.8127 m
Massa do íon: 3.3863e-25 kg
Massa do íon: 203.9334 u


# Resolução Física do Problema do Espectrômetro de Massa

## Descrição do Problema
Um íon de massa $m$ e carga $q$ é acelerado por uma diferença de potencial $V$. Em seguida, ele entra numa região com campo magnético uniforme $\mathbf{B}$ perpendicular à sua velocidade, descrevendo uma trajetória semicircular de raio $r$ até atingir um detector a uma distância $x=2r$ da entrada.

---

## Energia Cinética e Velocidade

Ao ser acelerado por $V$, o trabalho da força elétrica vira energia cinética:
$$
q\,V \;=\; \tfrac12\,m\,v^2
\quad\Longrightarrow\quad
v = \sqrt{\frac{2\,q\,V}{m}}.
$$

---

## Força Centrípeta no Campo Magnético

No campo $\mathbf{B}\perp\mathbf{v}$, a força magnética fornece a força centrípeta:
$$
F_\text{B} = q\,v\,B
\quad\text{e}\quad
F_\text{centrípeta} = \frac{m\,v^2}{r}.
$$
Igualando:
$$
q\,v\,B = \frac{m\,v^2}{r}
\quad\Longrightarrow\quad
r = \frac{m\,v}{q\,B}.
$$
Substituindo $v$ (isto é, $v^2=2qV/m$):
$$
r = \frac{m}{qB}\,\sqrt{\frac{2qV}{m}}
\;\Longrightarrow\;
r^2 = \frac{m\,2qV}{(qB)^2}
\;\Longrightarrow\;
m = \frac{q\,B^2\,r^2}{2\,V}.
$$

---

## Cálculo Numérico

1. **Dados**  
   - $B = 80\,000\ \mathrm{mT} = 0{,}080\ \mathrm T$  
   - $V = 1000{,}0\ \mathrm V$  
   - $q = 1{,}6022\times10^{-19}\ \mathrm C$  
   - $x = 1{,}6254\ \mathrm m\implies r = \tfrac{x}{2} = 0{,}8127\ \mathrm m$

2. **Massa em kg**  
   $$
   m = \frac{(1{,}6022\times10^{-19})\,(0{,}080)^2\,(0{,}8127)^2}{2\times1000}
     = 3{,}3863\times10^{-25}\ \mathrm{kg}.
   $$

3. **Conversão para unidades atômicas**  
   $$
   1\ \mathrm u = 1{,}6605\times10^{-27}\ \mathrm{kg}
   \quad\Longrightarrow\quad
   m = \frac{3{,}3863\times10^{-25}}{1{,}6605\times10^{-27}}
     = 203{,}93\ \mathrm u.
   $$

---

## Resultado Final

$$
\boxed{m \approx 3{,}39\times10^{-25}\ \mathrm{kg}
\quad\Longleftrightarrow\quad
m \approx 204\ \mathrm{u}.}
$$
